### PR TITLE
[1.7.10] Removed unfinished features

### PR DESCRIPTION
**Pull request by KorDum**
Removed unused Honey Crystal
Removed unfinished kitchen machines
Removed Alcohol

### CODE REVIEW

**Comment by KorDum** [comment]
@mezz do you want to publish on Curse the version first without this PR?

**Comment by lex1975** [comment]
Please don't remove item extrabees.item.scentedGear
Please don't remove unused liquids:
latex, sap, liquidnitrogen, poison, acid

These items and liquids can be used to create new recipes through CraftTweaker.

For example, I use this script:

// Squeezer. Create latex.
mods.forestry.Squeezer.addRecipe(<liquid:latex> * 1000, <Forestry:woodPulp:0> % 100, [<minecraft:log2:0>,<minecraft:log2:1>,<minecraft:log:2>], 20); 

// Foresry alveary block. Complication of recipe.
recipes.remove(<Forestry:alveary:0>);
mods.forestry.Carpenter.addRecipe(<Forestry:alveary:0>,
[[<Forestry:craftingMaterial:6>,<ExtraTrees:misc:3>,<Forestry:craftingMaterial:6>],
[<Forestry:craftingMaterial:6>,<Forestry:impregnatedCasing:0>,<Forestry:craftingMaterial:6>],
[<Forestry:craftingMaterial:6>,<ExtraBees:misc:0>,<Forestry:craftingMaterial:6>]],
<liquid:latex>*1000,20,null);

// ExtraTrees. Proven Gear. Complication of recipe.
recipes.remove(<ExtraTrees:misc:3>);
mods.forestry.Carpenter.addRecipe(<ExtraTrees:misc:3>,
[[null,<Forestry:oakStick:0>,null],
[<Forestry:oakStick:0>,<Forestry:beeswax:0>,<Forestry:oakStick:0>],
[null,<Forestry:oakStick:0>,null]],
<liquid:latex>*1000,20,null);

// Forestry Proven frame
recipes.addShaped(<Forestry:frameProven:0>,
[[<Forestry:oakStick:0>,<Forestry:oakStick:0>,<Forestry:oakStick:0>],
[<Forestry:oakStick:0>,<ExtraTrees:misc:3>,<Forestry:oakStick:0>],
[<Forestry:oakStick:0>,<Forestry:oakStick:0>,<Forestry:oakStick:0>]]);

// Foretry Proven grafter
recipes.addShaped(<Forestry:grafterProven:0>,
[[null,null,<ExtraTrees:misc:3>],
[null,<Forestry:oakStick:0>,null],
[<ExtraBees:misc:0>,null,null]]);

// Input Coolant Fluid - Binnie Liquid Nitrogen 0.0161mB/t).
mods.buildcraft.Fuels.addCombustionEngineCoolant(<liquid:liquidnitrogen>,2.0);

**Comment by KorDum** [comment]
> Please don't remove item extrabees.item.scentedGear
> Please don't remove unused liquids:
> latex, sap, liquidnitrogen, poison, acid

Don't worry, it was not deleted.
You may build mod from PR source and check it self.

File: README.md [hunk 1]
```diff
@@ -2,6 +2,13 @@
 
 All builds can be found on [Jenkins](https://ci.bymarcin.com/job/Binnie-Mods-MC1.7.10/)
 
+### Warning
+
+This is an unofficial fork of mod.
```
Comment (mezz) [inline README.md]: This fork is official, Binnie gave me/us the rights to do it.

**Review by mezz — CHANGES_REQUESTED** [review]

**Comment by mezz** [comment]
I think this should be included in the upcoming release, I can mark it as an alpha for a week.

File: README.md (hunk 1 above)
Comment (KorDum) [reply README.md]: Fixed :)

**Review by mezz — APPROVED** [review]

**Pull request merged**